src/form.js

```diff
@@ -4,7 +4,7 @@ define([
     'jquery',
     'vellum/tree',
     'vellum/logic',
-    'vellum/xpath',
+    'vellum/bananas',
```

Comment:
not a fan of this change. it's less meaningful to a new person looking at the code